runtime/compilers/rillv1/parse_api.go

```diff
@@ -0,0 +1,53 @@
+package rillv1
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
+)
+
+// APIYAML
+type APIYAML struct {
+	SQL        string         `yaml:"sql"`
```

Comment:
As an exception, the YAML shouldn't have a `SQL` property because that's already parsed by the `parseNode` function and can be accessed on `node.SQL`. This will automatically enable having SQL in a separate `.sql` file.